### ISSUE TITLE
Where did all these focus outlines come from?

### ISSUE THREAD

**Issue by allpwrfulroot** [IBM/carbon-components-react#1446]
## Help with managing focus outlines

For some reason, in my development environment I get a persistent focused state for any component I interact with: buttons, checkboxes, etc. This behavior isn't seen when interacting with the same components on either the official documentation or the official Storybook.

Usually, my projects follow the helpful hack outlined [here](https://hackernoon.com/removing-that-ugly-focus-ring-and-keeping-it-too-6c8727fefcd2), so that users interact with things focus-free by default but a single tab will restore it:

```
body:not(.user-is-tabbing) button:focus,
body:not(.user-is-tabbing) input:focus,
body:not(.user-is-tabbing) select:focus,
body:not(.user-is-tabbing) {
  outline: none;
}
```

However, nothing I've tried works for removing the focus outline from Carbon's checkbox, radio, etc. I must be missing something fundamental, since I don't see this issue with any published implementation. 

Has anyone else encountered this issue and found a fix?

**Comment by asudoh** [comment]
Hi 👋 our checkbox/radio UI is achieved by a pseudo element in `<label>`, etc. instead of `<input>` itself. You can see it by inspecting the DOM. Hope it helps!
